test/test_dtype.py

```diff
@@ -3,7 +3,9 @@
 from tinygrad.helpers import getenv, DType, DEBUG
 from tinygrad.lazy import Device
 from tinygrad.tensor import Tensor, dtypes
+from tinygrad.runtime.ops_cpu import match_types
 from extra.utils import OSX
+import torch
```

Comment:
Why?

Reply:
I had unit tests for match_types, but I removed them because they were breaking the CI in weird ways. I forgot to remove the imports while working on fixing the CI